devcycle_python_sdk/managers/config_manager.py

```diff
@@ -89,4 +89,4 @@ def run(self):
 
     def close(self):
         self._polling_enabled = False
-        self.join(timeout=5.0)
+        self.join(timeout=1)
```

Comment:
I'm waffling on whether the join is really necessary since the threads are daemons anyway -- it does make the clean up cleaner though

Reply:
If it's got a timeout, seems harmless. And if we can avoid broken looking logs / exceptions occasionally popping up by shutting down cleanly, probably an improvement.